### ISSUE TITLE
Neural Network for Multiple Object Recognition

### ISSUE THREAD

**Issue by kylehl** [UCI-UAVForge/Avionics#80]
Prerequisite Issues: None
Followup Issues: TBD
Related Issues: TBD

### Summary
Create a neural network that can identify multiple objects in a given image.

### Terminal Objective
* Neural network that receives images from the UAV, processes them, sends relevant avoidance instructions back to UAV and displays recognition data to user.

### Enabling Objectives
* Complete a simple convolutional neural network.
* Hyperparameter optimization.
* Move to Linux and try using the machine learning framework, or try rebuilding the existing neural network architecture
* Achieve multiple object recognition.
* Image captioning by using recurrent neural network.

**Comment by albertml123** [comment]
Windows code having problem with dimension. Having trouble applying padding.
Starting to write code using Tensorflow on Linux.
Looking for a image set with at least dimension 128x128.

**Comment by albertml123** [comment]
Cifar-10 I am getting around 85% accuracy
Cifar-100 I am getting around 50% accuracy (not fully trained)
My goal is to get a higher accuracy rate in Cifar-100 by adding more layers and experiment in the inputs, like upscaling the images.

**Comment by albertml123** [comment]
cifar-100 code uploaded to my fork
looking into how to caption image with recurrent neural network
learn how to code recurrent neural network on tensorflow

**Comment by albertml123** [comment]
This week try to code object avoidance with optical flow algorithm

**Comment by kylehl** [comment]
Task suspended until further notice due to low priority.
